lib/index.js

```diff
@@ -143,7 +143,9 @@ module.exports.registerEventHandler = registerEventHandler;
 module.exports.add = add;
 module.exports.remove = remove;
 module.exports.skip = TrackPlayer.skip;
+module.exports.getQueue = TrackPlayer.getQueue;
 module.exports.skipToNext = TrackPlayer.skipToNext;
+module.exports.clearQueue = TrackPlayer.clearQueue;
```

Comment:
Added this one as well. It will clear the queue but not stop the currently playing track.